scripts/mutate.py

```diff
@@ -71,7 +71,7 @@ class MutationType(Enum):
         "leq_ratio_ratio",
         "geq_ratio_ratio",
         "gt_ratio_ratio",
-        "eq_ratio_ratio",
+        # there is no eq_ratio_ratio
```

Comment:
Actually there is `eq_ratio_ratio`, but it's only useful for the tests so indeed there's no reason to consider it at all when mutating :+1: In fact, now that I look at it, this is true for all the comparing functions in `ratio.ml`: `leq_ratio_ratio`, `geq_ratio_ratio`, `lt_ratio_ratio`, `gt_ratio_ratio`, and `eq_ratio_ratio`; I don't think there's any merit in trying to mutate any of them.

Reply:
Ooh, right. I think I was grepping them under the `src/` so I thought they were unused. You are right about no point testing them, but maybe it would be useful as mutation targets. And at worst, they will be unused.

I will definitely fix the comment, but I think I should keep the `eq_ratio_ratio` there.

Reply:
Yeah, fair point, you can leave all of them there, just leave a comment that we don't really expect those to fire :+1: